dashboard/lib/policies/child_account.rb

```diff
@@ -65,7 +53,7 @@ def self.compliant?(user)
   # state missing
   # We use Colorado as it is the only start date we have for now
   def self.user_predates_state_collection?(user)
-    user.created_at < STATE_POLICY['CO'][:start_date]
+    user.created_at < state_policy(user)[:start_date]
```

Comment:
Actually we don't want to do this because the user might not have their state yet. We use this colorado date because it happens to also be the date we us-state collection. We should actually create a separate const....